src/application.coffee

```diff
@@ -121,6 +134,10 @@ application = {}
 application.UpdatesLockedError = UpdatesLockedError
 application.localMode = false
 
+application.healthy = ->
+	timeSinceLastCycle = (process.hrtime()[0] - updateStatus.lastFullUpdateCycle) * 1000
```

Comment:
Why not `Date.now()`?

Reply:
Because process.hrtime will not change if ntp changes the clock